src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -79,7 +80,7 @@ export const CollectorProfileSaves2RouteFragmentContainer = createFragmentContai
       fragment CollectorProfileSaves2Route_me on Me {
         defaultSaves: collection(id: "saved-artwork") {
           internalID
-          ...SavesItem_item
+          ...ArtworkListItem_item
         }
 
         otherSaves: collectionsConnection(
```

Comment:
**question**: do we also need to rename `defaultSaves` and `otherSaves`? For example, to`allSavesArtworkList` or `defaultArtworkList` and `otherArtworkLists` or `customArtworkLists`

Reply:
> My view is that they all should be renamed so that we have as much consistency as possible.

Got it. I'll do it

Reply:
> like the fragment name CollectorProfileSaves2Route_me

@anandaroop I'll rename it to `ArtworkListsRoute_me`. is it ok?

Reply:
I guess Relay requires some strictness around naming with respect to components? I leave it up to you the best way / best sequence for doing this, but yes that name sounds fine to me.